pxtlib/tilemap.ts

```diff
@@ -1020,20 +1022,22 @@ namespace pxt {
                         }
                     }
                 }
-            }
 
-            for (const tm of getTilemaps(pack.parseJRes())) {
-                this.state.tilemaps.add({
-                    internalID: this.getNewInternalId(),
-                    type: AssetType.Tilemap,
-                    id: tm.id,
-                    meta: {
-                        // For tilemaps, use the id as the display name for backwards compat
-                        displayName: tm.displayName || tm.id
-                    },
-                    data: decodeTilemap(tm, id => this.resolveTile(id))
-                })
+                for (const tm of getTilemaps(dep.parseJRes())) {
+                    this.state.tilemaps.add({
+                        internalID: this.getNewInternalId(),
+                        type: AssetType.Tilemap,
+                        id: tm.id,
+                        meta: {
+                            // For tilemaps, use the id as the display name for backwards compat
+                            displayName: tm.displayName || tm.id,
+                            package: dep.id
```

Comment:
Ah hm, I meant just adding in the package field so it exists (like `package: pack.id`), not necessarily expanding it to read in tilemaps from extensions right now; it probably shouldn't do any harm as if they had tilemaps we'd already error on the generated files conflicting but I'm not certain / don't think there's a benefit to pulling them in at this time; @riknoll any opinion here?

Reply:
yeah, i'd stick with pack.parseJRes()


if we're going to change that behavior, let's do it in a different pr